doc/source/whatsnew/v1.2.1.rst

```diff
@@ -35,6 +35,10 @@ Fixed regressions
 - Fixed regression in comparisons between ``NaT`` and ``datetime.date`` objects incorrectly returning ``True`` (:issue:`39151`)
 - Fixed regression in :func:`pandas.testing.assert_index_equal` raising ``TypeError`` with ``check_order=False`` when :class:`Index` has mixed dtype (:issue:`39168`)
 
+We have reverted a commit that resulted in several plotting related regressions in pandas 1.2 (:issue:`38969`, :issue:`38736`, :issue:`38865`, :issue:`38947` and :issue:`39126`)
+
```

Comment:
```suggestion
We have reverted a commit that resulted in several plotting related regressions in pandas 1.2 (:issue:`38969`, :issue:`38736`, :issue:`38865`, :issue:`38947` and :issue:`39126`).
```

(I think it is fine to have it one paragraph instead of two, so it's clear it's one topic)

Reply:
if its one paragraph could maybe include in the list instead?

Reply:
That's fine for me as well, but by not including it in the list but having it as separate paragraph, it stands out a bit more, so I think that's also fine.

Reply:
no problem. This is fine as is.